result/dhcpv6serverleases.yaml

```diff
@@ -0,0 +1,76 @@
+components:
+  schemas:
+    Dhcpv6Lease.State.Request:
+      description: >-
+        The request to retrieve DHCP Server host allocated status.
+      type: object
+      properties:
+        dhcp_server_names:
+          description: >-
+            The names of DHCPv6 server to return results for.
+            An empty list will return results for all DHCPv6 servers.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Device.Dhcpv6server/properties/name"
+          x-field-uid: 1     
+    Dhcpv6Leases.State:
+      description: >-
+       Lease information of DHCP Server.
+      type: object
+      properties:
+        dhcp_server_name:
+          description: >-
+            The name of a DHCP Server.
+          type: string
+          x-field-uid: 1
+        leases:
+          type: array
+          items:
+            $ref: '#/components/schemas/Dhcpv6ServerLease.State'
+          x-field-uid: 2
+    Dhcpv6ServerLease.State:
+      description: >-
+        IPv6 unicast prefix.
+      type: object
+      properties:
```

Comment:
Is the prefix length and gateway not something that is learned  ? If not , how would we resolve dst mac ?

Reply:
I think we won't keep the client mac  in  learned info but should keep the client's Mac in the dB and provide it in the traffic query